src/pages/examSetter/Proctoring.tsx

```
@@ -0,0 +1,136 @@
+        title={`Candidates for ${selectedExam?.name}`}
+        visible={isModalVisible}
+        onCancel={() => setIsModalVisible(false)}
+        footer={null}
+      >
+        <List
+          itemLayout="horizontal"
+          dataSource={candidates}
+          renderItem={(candidate: Candidate) => (
+            <List.Item
+              actions={[
+                <Tooltip title="View Candidate">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)